coverage/files.py

```diff
@@ -316,7 +316,7 @@ def sep(s: str) -> str:
     (r"\*\*+[^/]+", None),          # Can't have **x
     (r"\*\*/\*\*", None),           # Can't have **/**
     (r"^\*+/", r"(.*[/\\\\])?"),    # ^*/ matches any prefix-slash, or nothing.
-    (r"/\*+$", r"[/\\\\].*"),       # /*$ matches any slash-suffix.
+    (r"/\*\*+$", r"[/\\\\].*"),       # /**$ matches any slash-suffix.
```

Comment:
@nedbat at first I thought I'd just send a test case demonstrating the problem, but then, I thought it should be easy to fix, so I've added this change as well. I don't know if there was a motivation for things to work as they do, but hopefully this can improve the behavior and match it with what's documented…